io/plugins/eu.esdihumboldt.hale.io.validation/plugin.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension
+         point="eu.esdihumboldt.hale.io.action">
+      <action
+            categoryName="Validator configurations"
+            id="eu.esdihumboldt.hale.io.validation.read"
+            name="Load validator configuration"
+            resourceName="validator configuration"
+            type="eu.esdihumboldt.hale.io.validation.ValidationConfigurationReaderImpl">
+      </action>
+      <advisor
+            action="eu.esdihumboldt.hale.io.validation.read"
+            class="eu.esdihumboldt.hale.io.validation.ValidatorConfigurationImportAdvisor"
+            id="eu.esdihumboldt.hale.io.validation.service">
+      </advisor>
+   </extension>
+   <extension
+         point="eu.esdihumboldt.util.resource">
+      <resourceType
+            id="validatorConfiguration"
+            name="Validator Configuration">
+      </resourceType>
+   </extension>
+   <extension
+         point="eu.esdihumboldt.hale.io.provider">
+      <provider
+            class="eu.esdihumboldt.hale.io.validation.ValidationConfigurationReaderImpl"
+            id="eu.esdihumboldt.hale.io.validation.reader"
+            name="Validator configuration">
+         <contentType
+               ref="eu.esdihumboldt.hale.io.schematron.schema">
+         </contentType>
+      </provider>
+      <provider
+            class="eu.esdihumboldt.hale.io.validation.ProjectValidator"
+            id="eu.esdihumboldt.hale.io.validation.projectvalidator"
+            name="Project validator">
+         <contentType
```

Comment:
That might be a way. I think at least for text files there is a content type defined by Eclipse, could be that the XML content type extends it. But I would leave it as it is for now, and leave this for later when the problem actually comes up.